census/src/main/java/io/grpc/census/CensusStatsModule.java

```diff
@@ -67,6 +68,7 @@
  * starts earlier than the ServerCall.  Therefore, only one tracer is created per stream/call and
  * it's the tracer that reports the summary to Census.
  */
+@VisibleForTesting
```

Comment:
It seems this should be marked `@Internal` or similar? `@VisibleForTesting` does not really limit users.

Reply:
Changed to `@Internal`.

Reply:
`CensusStatsModule` and `CensusTracingModule` are now package-private.